### DISCUSSION TITLE
How to set decimal point?

### DISCUSSION THREAD

**Discussion by carslen** [jomjol/AI-on-the-edge-device#2051] [Q&A] [answered]
Hi community,
after my first steps with AI on the edge device reading my water meter and adding it to my Home Assistant, today I started to read my gas meter as well. In general, the numbers are identified correctly, but how to set the decimal point? Will it recognized based on the red background, or do I have to edit some setting?

<img width="956" alt="image" src="https://user-images.githubusercontent.com/28647218/219956924-8bf2c497-5aba-4f0b-8e52-7bf560890059.png">

Cheers Carsten

**Comment by caco3** [comment, marked as answer]
see the config and https://jomjol.github.io/AI-on-the-edge-device-docs/Parameters/#PostProcessing-NUMBER.DecimalShift

**Comment by carslen** [comment]
Sometimes it is too obvious. :)
